### PR TITLE
Use Standard Style

### PR DESCRIPTION
**Pull request by maxbeatty**
When I saw #48, I looked to [the travis job](https://travis-ci.org/jsperf/jsperf.com/builds/84592207) to see what didn't pass only to find:

```
$ npm test
> jsperf.com@2.0.0 test /home/travis/build/jsperf/jsperf.com
> lab -t 100 -c -L -r lcov -o lcov.info
npm ERR! Test failed.  See above for more details.
```

Not very helpful so I ran the tests locally adding a second reporter of "console" to `stdout`. Sure enough the test was broken, but there were also a lot of linting errors. Some mentioned outdated rules and at that point, I decided to pick a more widely used style that others maintain.

_I don't have any strong feelings towards [Standard](http://standardjs.com/). We could use another rule set. I mostly just want to easily extend a config with eslint._

And with that... I edited every file in the project... but now the tests pass and you can easily see what's wrong when you run the _standard_ `npm test`

```
$ npm test

> jsperf.com@2.0.0 test /Users/max/Sites/jsperf.com
> lab -t 100 -c -L -r lcov -o lcov.info -r console -o stdout


  
  ..................................................
  ..................................................
  ..................................................
  .................

167 tests complete
Test duration: 2824 ms
No global variable leaks detected
Coverage: 100.00%
Linting results: No issues
```

I'll add some comments on specific parts that changed beyond style

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,27 +1,3 @@
...
-    "space-in-parens": [2, "never"],
-    "space-infix-ops": 2,
-    "spaced-line-comment": [2, "always"],
-    "space-return-throw-case": 2
-  },
-  "ecmaFeatures": {
-    "blockBindings": true,
-    "templateStrings": false
-  }
+  "extends": "standard"
```
Comment (maxbeatty) [inline .eslintrc]: use standard style instead of maintaining our own `.eslintrc`

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-iojs-v2.5.0
+v4.1.2
```
Comment (maxbeatty) [inline .nvmrc]: use latest node version

File: config.js [hunk 3]
```diff
@@ -1,27 +1,25 @@
-"use strict";
```
Comment (maxbeatty) [inline config.js]: I don't remember why I had `"use strict"` everywhere but should be safe to remove now

File: package.json [hunk 4]
```diff
@@ -4,14 +4,12 @@
...
   "main": "index.js",
   "scripts": {
+    "clean": "rm npm-debug.log lcov.info coverage.html",
     "lint": "lab -d -L",
     "setup": "node setup",
     "start": "nodemon --ext js,hbs server.js",
-    "test": "lab -t 100 -c -L -r lcov -o lcov.info",
-    "test-cov": "lab -t 100 -c -L",
-    "test-cov-html": "lab -r html -o coverage.html && open coverage.html",
-    "test-fast": "lab",
```
Comment (maxbeatty) [inline package.json]: simplified the test commands to encourage people to always run all the tests with linting and coverage

File: package.json [hunk 5]
```diff
@@ -49,19 +47,17 @@
     "yar": "^3.0.3"
   },
   "devDependencies": {
-    "boom": "^2.8.0",
```
Comment (maxbeatty) [inline package.json]: removed some dup dependencies

**Comment by maxbeatty** [comment]
buried inside this is an update to this test that should allow it to pass in multiple locales to resolve #48

```js
Code.expect(relativeDate(d)).to.equal('on ' + d.toLocaleDateString())
```

ready for review and can be merged if there's no issues :shipit:

**Comment by jdalton** [comment]
I'd prefer the standard semicolon variant fwiw.

**Comment by mathiasbynens** [comment]
@jdalton Me too. But if @maxbeatty prefers it this way, I can live with it.

Either way, :+1: to setting a strict coding standard and automatically checking it!

File: config.js (hunk 3 above)
Comment (mathiasbynens) [inline config.js]: Maybe it was because of “SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode”?

**Comment by maxbeatty** [comment]
I don't have a strong opinion regarding semicolons. If there's a tool or trick to add them all back (anything but one at a time :sweat_smile:), I'd be happy to do it

File: config.js (hunk 3 above)
Comment (maxbeatty) [inline config.js]: That makes sense. I did change a few `let`s to `var`s for that reason

**Comment by jdalton** [comment]
Rock then [semistandard](https://github.com/Flet/semistandard) it up. Anything to avoid supporting `standard` really.

**Comment by maxbeatty** [comment]
Thanks to [semistandard-format cli](https://github.com/ricardofbarros/semistandard-format) adding back semicolons was pretty painless. Is there anything else to address?

**Comment by jdalton** [comment]
Can the commits be squished where approp.

**Comment by maxbeatty** [comment]
Is that something we're doing now? If so, will you add directions in `CONTRIBUTING.md` and go ahead with squishing as an example of what's appropriate?

**Comment by jdalton** [comment]
Hmm.. they're a little out of order, we can skip the squish.

**Pull request merged**